lib/publiccloud/acr.pm

```diff
@@ -20,8 +20,8 @@ use utils;
 has security_token => undef;
 has key_id => undef;
```

Comment:
```suggestion
has key_id => sub { get_var('PUBLIC_CLOUD_KEY_ID') };
```

Reply:
In this PR will be about only provider (leaving k8s_provider and childs to an other PR)